.travis.yml

```diff
@@ -89,20 +118,27 @@ matrix:
         - ./test/setup_contracts.sh
       script:
         - docker-compose -f docker-compose.yml -f docker-compose.binary.yml up -d driver graph-listener
-        - ./test/e2e-tests-deposit-withdraw.sh
-        - ./test/restart-containers.sh
         - ./test/e2e-tests-auction.sh
         - ./test/restart-containers.sh
         - ./test/e2e-tests-standing-order.sh
       after_failure:
         - docker-compose logs
-    - language: generic
+    - language: rust
```

Comment:
We should clean up the travis file in the end. All this stuff can become the default.